gettsim/transfers/grunds_im_alter.py

```diff
@@ -0,0 +1,338 @@
+        :ref:`grunds_ia_params <grunds_ia_params>`.
+    Returns
+    -------
+
+    """
+
+    # Consider Elterngeld that is larger than 300
+    anrechenbares_elterngeld_m = (
+        elterngeld_m - grunds_ia_params["elterngeld_anr_frei"]
+    ).clip(lower=0)
+
+    # Income
+    total_income = (
+        anrechenbares_erwerbs_eink_grunds_ia_m
+        + anrechenbare_staatl_rente_grunds_ia_m
+        + anrechenbare_prv_rente_grunds_ia_m
+        + sonstig_eink_m
+        + vermiet_eink_m
+        + kapital_eink_minus_pauschbetr / 12
+        + anrechenbares_elterngeld_m
+    )
+
+    # subtract taxes and social security contributions
+    out = (
+        total_income
+        - tu_id.replace((eink_st_tu / anz_erwachsene_tu) / 12)
+        - tu_id.replace((soli_st_tu / anz_erwachsene_tu) / 12)
+        - sozialv_beitr_m
+    ).clip(lower=0)
+
+    return out
+
+
+def anrechenbares_erwerbs_eink_grunds_ia_m(
+    bruttolohn_m: FloatSeries,
+    eink_selbst_m: FloatSeries,
+    arbeitsl_geld_2_params: dict,
+    grunds_ia_params: dict,
+) -> FloatSeries:
+    """Calculate earnings relevant for Grundsicherung of Grundsicherung im Alter.
+
+    Legal reference: § 82 SGB XII Abs. 3
+
+    Note: Freibeträge for income are currently not considered
+    Note: the cap at 1/2 of Regelbedarf was only introduced in 2006 (which is currently
+    not implemented): https://www.buzer.de/gesetz/3415/al3764-0.htm
+
+    Parameters
+    ----------
+    bruttolohn_m
+        See basic input variable :ref:`bruttolohn_m <bruttolohn_m>`.
+    eink_selbst_m
+        See basic input variable :ref:`eink_selbst_m <eink_selbst_m>`.
+    arbeitsl_geld_2_params
+        See params documentation :ref:`arbeitsl_geld_2_params <arbeitsl_geld_2_params>`.
+    grunds_ia_params
+        See params documentation :ref:`grunds_ia_params <grunds_ia_params>`.
+    Returns
+    -------
+
+    """
+    earnings = bruttolohn_m + eink_selbst_m
+
+    # Can deduct 30% of earnings (but no more than 1/2 of regelbedarf)
+    earnings_after_max_deduction = earnings - arbeitsl_geld_2_params["regelsatz"][1] / 2
+    earnings = ((1 - grunds_ia_params["erwerbs_eink_anr_frei"]) * earnings).clip(
+        lower=earnings_after_max_deduction
+    )
+
+    return earnings
+
+
+def anrechenbare_prv_rente_grunds_ia_m(
+    prv_rente_m: FloatSeries, arbeitsl_geld_2_params: dict, grunds_ia_params: dict,
+) -> FloatSeries:
+    """Calculate public pension earnings relevant for Grundsicherung of Grundsicherung
+    im Alter.
+
+    Legal reference: § 82 SGB XII Abs. 4
+
+    Parameters
+    ----------
+    prv_rente_m
+        See basic input variable :ref:`prv_rente_m <prv_rente_m>`.
+    arbeitsl_geld_2_params
+        See params documentation :ref:`arbeitsl_geld_2_params <arbeitsl_geld_2_params>`.
+    grunds_ia_params
+        See params documentation :ref:`grunds_ia_params <grunds_ia_params>`.
+
+    Returns
+    -------
+
+    """
+    deducted_rent = piecewise_polynomial(
```

Comment:
```suggestion
    prv_rente_m_amount_exempt = piecewise_polynomial(
```

rent ≠ pensions

And this way it really should be clear what is meant, even if there are a couple of extra parameters.